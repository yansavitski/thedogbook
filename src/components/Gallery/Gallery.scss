@import 'src/global';

.Gallery {
  display: grid;
  grid-gap: $baseOffset/2;
  grid-template-columns: repeat(auto-fit, 200px);
  justify-content: center;
}
