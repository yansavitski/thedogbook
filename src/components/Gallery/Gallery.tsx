import './Gallery.scss'

import classNames from 'classnames'
import React from 'react'

interface GalleryProps {
  children: React.ReactElement
  className?: string
}

export const Gallery: React.FunctionComponent<GalleryProps> = ({
  className,
  children,
}: GalleryProps) => {
  return (
    <section className={classNames('Gallery', className)}>{children}</section>
  )
}
