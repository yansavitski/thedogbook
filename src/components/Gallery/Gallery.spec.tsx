import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import React from 'react'

import { Gallery } from './Gallery'

describe('<Gallery />', () => {
  it('renders correctly', () => {
    expect.assertions(1)

    const tree = shallow(
      <Gallery className="wrapperGalleryClass">
        <span>text</span>
      </Gallery>,
    )

    expect(toJson(tree)).toMatchSnapshot()
  })
})
