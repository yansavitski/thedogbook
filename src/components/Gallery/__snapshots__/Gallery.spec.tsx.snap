// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Gallery /> renders correctly 1`] = `
<section
  className="Gallery wrapperGalleryClass"
>
  <span>
    text
  </span>
</section>
`;
